src/components/picker/index.tsx

```diff
@@ -13,6 +13,8 @@ export class SmoothlyPicker {
 	@Element() element: HTMLElement
 	@State() isOpen: boolean
 	@State() empty: boolean
+	@Prop({ reflect: true, attribute: "data-disabled" }) disabled = false
+	@Prop({ reflect: true, attribute: "data-readonly" }) readonly = false
```

Comment:
It is not semantically correct HTML to *__not__* prefix custom attributes with `data-*` [MDN docs](https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes ). For example `<body foo=""></body>` gives the error `Error: Attribute foo not allowed on element [body](https://html.spec.whatwg.org/multipage/#the-body-element) at this point.` when validating with an [HTML validator](https://validator.w3.org/)

Reply:
Nevermind. This is not relevant to custom elements aparently.

Reply:
Well, I cloud argue either that:

1. `<smoothly-picker disabled readonly />` are not custom attributes since we are the authors of the `smoothly-picker` element and we define what the standard attributes are. Whereas the element `body` is not decided by us, so `foo` would be a custom element.

or 

2. If any reflected property that we add is custom than we should always add `data-*` to all our reflected properties.